### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by tobybot11**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/tobybot11/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
-# ReviewNinja-Welcome
\ No newline at end of file
+# ReviewNinja-Welcome
+
```
Comment (tobybot11) [inline ReadMe.md:2]: no... thank you!

Comment (tobybot11) [inline ReadMe.md:2]: !fix

Comment (tobybot11) [inline ReadMe.md:2]: !fixed

**Pull request merged**